dependencies.gradle

```diff
@@ -0,0 +1,29 @@
+ext {
+  androidPlugin = 'com.android.tools.build:gradle:1.3.1'
+  minSdkVersion = 14
+  compileSdkVersion = 23
+  buildToolsVersion = '23.0.1'
+
+  ci = 'true'.equals(System.getenv('CI'))
+
+  kotlinPlugin = 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.0.0-beta-1038'
+  kotlinStdlib = 'org.jetbrains.kotlin:kotlin-stdlib:1.0.0-beta-1038'
+
+  supportAnnotations = 'com.android.support:support-annotations:23.1.0'
```

Comment:
Would be nice to declare the version of support libs as a separate constant so you could change it in one place and it will be applied to all support libs (magic)!